src/Storages/tests/gtest_background_low_priority_processing_pool.cpp

```diff
@@ -0,0 +1,129 @@
+#include <gtest/gtest.h>
+#include <Interpreters/Context.h>
+#include <Common/Config/ConfigProcessor.h>
+#include <Common/tests/gtest_global_context.h>
+#include <Storages/MergeTree/BackgroundProcessingPool.h>
+#include <iostream>
+#include <Poco/Util/Application.h>
+#include <daemon/BaseDaemon.h>
+
+DB::BackgroundProcessingPoolTaskResult checkLow(std::vector<bool> & checks, int i, std::mutex & mut)
+{
+    int policy;
+    sched_param param;
+    if (pthread_getschedparam(pthread_self(), &policy, &param))
+    {
+        std::lock_guard<std::mutex> guard(mut);
+        checks[i] = false;
+
+        return DB::BackgroundProcessingPoolTaskResult::ERROR;
```

Comment:
There really is no point in returning anything other than SUCCESS here - our checks complete either way, and status ERROR signals to workers that task needs to be retried